package.json

```diff
@@ -49,11 +49,11 @@
         "@floating-ui/react-dom": "0.6.0",
         "@floating-ui/react-dom-interactions": "0.6.3",
         "@mdi/react": "1.5.0",
+        "@ngard/tiny-isequal": "1.1.0",
```

Comment:
how tiny are we talking about here? should we pull it in as a util to reduce external dependencies?